spectragram.js

```diff
@@ -156,10 +160,7 @@ if ( typeof Object.create !== "function" ) {
 
 					$image = $( "<img>", {
 						alt: imageCaption,
-						attr: {
-							height: imageHeight,
-							width: imageWidth
-						},
+						style: "width: " + (WidthSetting == null ? imageWidth + "px" : WidthSetting) + "; height: " + (HeightSetting == null ? imageHeight + "px": HeightSetting),
```

Comment:
This line is above 120 characters, consider splitting it to make it more readable